python/lsst/verify/bin/jobReporter.py

```diff
@@ -106,14 +105,17 @@ def run(self):
             datasetRefs = list(self.registry.queryDatasets(dataset,
                                collections=self.collection))
             for ref in datasetRefs:
+                names = list(ref.dataId.names)
```

Comment:
I recommend calling `sorted`, just in case the iteration order for `names` is not consistent between datasetRefs. Otherwise metric values from the same data ID may be assigned different keys.

Reply:
Good catch.  I just completely missed that this was done in the original (even with the comment)